stdlib/buffer.gr

```diff
@@ -414,33 +414,34 @@ export let rec addString = (string, buffer) => {
  */
 @disableGC
 export let rec addStringSlice = (start, length, string, buffer) => {
-  Memory.incRef(WasmI32.fromGrain(String.slice))
   Memory.incRef(WasmI32.fromGrain(start))
   Memory.incRef(WasmI32.fromGrain(length))
   Memory.incRef(WasmI32.fromGrain(string))
+
+  Memory.incRef(WasmI32.fromGrain(String.slice))
```

Comment:
I'd probably move this back to the top where it was—the general pattern we've been using is incRefing the function before the arguments.